gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/validation/EndpointGroupsValidationServiceImpl.java

```diff
@@ -92,9 +103,36 @@ private void validateEndpointsMatchType(final EndpointGroup endpointGroup) {
         }
     }
 
+    private void validateHealthCheck(Service healthCheck) {
```

Comment:
the validation would be handled by the appropriate service when implemented.

Reply:
Yes, it's definitively the thing to do, but until then would you mind if I keep the empty methods ? As a reminder for the future.